<!-- 
FP: 30 Jan. 2013

Files tuned for ethzasl_icp_mapping commit 52d78c96c2cf2f58a400c649c561dbfec95e41a4
and for libpointmatcher commit e048c4d1180b1de005af072bf137ea4d8cf2b826
-->

<launch>
	<node name="Keyframe_node" type="Keyframe_node" pkg="slam" output="screen" >
		<!--remap from="/lidar/lidarfilteringnode/localmap" to="cloud_pcd" /-->
		<param name="icpConfig" value="$(find slam)/launch/icp_keyframe.yaml" />
		<param name="inputFiltersConfig" value="$(find slam)/launch/input_filters_keyframe.yaml" />
		
		
		
		<!--param name="useROSLogger" value="true" /-->
		<param name="minOverlap" value="0.5" /> 
		<!--param name="minReadingPointCount" value="1" /--> 
		<!--param name="minMapPointCount" value="1" /--> <!-- quick fix to have map published each time -->
		<!--param name="tfRefreshPeriod" value="0"/-->
	</node>

	
</launch>
